localstack/services/ses/provider.py

```diff
@@ -196,31 +216,36 @@ def get_identity_verification_attributes(
             VerificationAttributes=attributes,
         )
 
-    @handler("SendEmail")
+    @handler("SendEmail", expand=False)
     def send_email(
         self,
         context: RequestContext,
-        source: Address,
-        destination: Destination,
-        message: Message,
-        reply_to_addresses: AddressList = None,
-        return_path: Address = None,
-        source_arn: AmazonResourceName = None,
-        return_path_arn: AmazonResourceName = None,
-        tags: MessageTagList = None,
-        configuration_set_name: ConfigurationSetName = None,
+        request: SendEmailRequest,
     ) -> SendEmailResponse:
+        backend = get_ses_backend(context)
+        emitter = SNSEmitter(context)
+        for event_destination in backend.config_set_event_destination.values():
+            if not event_destination["Enabled"]:
+                continue
+
+            sns_destination_arn = event_destination.get("SNSDestination")
+            if not sns_destination_arn:
+                continue
+
+            emitter.emit_send_event(request, sns_destination_arn)
+            emitter.emit_delivery_event(request, sns_destination_arn)
```

Comment:
Should the events be sent after `call_moto` in case of an exception? I'm not sure, I don't know exactly how SES events work, but maybe they are not sent, or different if something is wrong?

Reply:
@thrau and I were discussing this earlier. I want to spend some time tomorrow trying to see what the error behaviour is for these, since now the services are coupled, errors in either SES or SNS must be handled in the right order. 

In general I think you're right, raising the SES errors first (i.e. sending events after `call_moto`) is the likely right behaviour here anyway :+1: